app/response/views.py

```diff
@@ -254,12 +256,25 @@ def response_email():
          "email_content": HTML
     }
 
-    :return: the HTML of the rendered template
+    :return: the json_resp and HTTP status code
+    Response Parameters:
+    - json_resp: json response data from process_email_template_request
+    - status_code: HTTP status code
+
+    Ex:
+    {
+        "json_resp": {
+            "error": "No changes detected',
+            "template": HTML rendered response template,
+            "header": "The following will be emailed to all associated participants:"
+        }
+        "status_code": 200
+    }
     """
     data = flask_request.form
     request_id = data['request_id']
-    email_template = process_email_template_request(request_id, data)
-    return email_template
+    json_resp, status_code = process_email_template_request(request_id, data)
+    return json_resp, status_code
```

Comment:
I should've realized earlier, but I think you can just return process_email_template(...)